main.tf

```diff
@@ -25,8 +28,9 @@ resource "azurerm_resource_group" "default" {
 # permissions.
 resource "azurerm_role_definition" "subscription" {
   for_each = toset(var.rsc_azure_features)
-  name        = "Rubrik Polaris SubRole ${each.key} - terraform - ${data.azurerm_subscription.current.subscription_id}"
-  description = "Rubrik Polaris Subscription role for ${each.key} - Terraform Generated"
+  
+  name        = "Rubrik Security Cloud SubRole ${each.key} - terraform - ${data.azurerm_subscription.current.subscription_id}"
+  description = "Rubrik Security Cloud Subscription role for ${each.key} - Terraform Generated"
```

Comment:
@joeharlan the name and description provided here match what RSC currently creates during the with and without OAuth workflows. Please confirm that you want to move away from this standard.

Reply:
@DamaniN The way I considered this versus how the "with OAuth" process creates the roles is they are mutually exclusive.  With the customers I have using these modules, each have their own naming standards.  These names were changed to match their preferences in every case.  This experience has led me to believe customers see these modules as prescriptive examples and not hard and fast rules.  Not one of them has used them as-is.  Instead, they have made significant modifications to suit their naming and IaC standards.  So long as they do not break things, this seems reasonable to permit.  If the customer is able to articulate which of their custom-named roles aligns to each CNP feature, then they should be able to effecively communucate with Rubrik Support.  Nothing about their names is hard-coded in the product from a functional standpoint.